programs/gpl_session/README.md

```
@@ -34,15 +36,15 @@ pub struct Instruction<'info> {
     // Session Tokens are passed as optional accounts
-    pub session_token: Option<Account<'info, SessionToken>>,
+    pub session_token: Option<Account<'info, SessionTokenV2>>,
 
     #[account(mut)]
     pub signer: Signer<'info>,
     .....
 }
 ```
 
-3. Add the `session_auth_or` macro to your instruction handler with fallback logic on who the instruction should validate the signer when sessions are not present and an appropirate ErrorCode. If you've used `require*!` macros in anchor_lang you already know how this works.
+3. Add the `session_auth_or` macro to your instruction handler with fallback logic on who the instruction should validate the signer when sessions are not present and an appropriate ErrorCode. If you've used `require*!` macros in anchor_lang you already know how this works.
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 3; Style: 1/1/1

(MD029, ol-prefix)